### PR TITLE
Add user images to each KudosNote

### PR DESCRIPTION
**Pull request by xevenheaven**
- This addresses the first task in #24

### CODE REVIEW

File: src/components/KudosNote.js [hunk 1]
```diff
@@ -6,10 +6,14 @@ module.exports = React.createClass({
 		let isoDate = this.props.date;
 		let fullDate = moment(isoDate).format('ddd, D MMM YYYY [at] h:mma');
 		let relativeDate = moment(isoDate).fromNow();
+		let userImg = `../images/${this.props.username}.png`;
 
 		return (
 			<div className="kudo">
-				<span className="kudo-author">{this.props.author}</span>
+				<div>
+					<img src={userImg}></img>
```
Comment (spinningarrow) [inline src/components/KudosNote.js]: `</img>`? o.O

File: src/components/KudosList.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ module.exports = React.createClass({
 
 			return (
 				<KudosNote author={author.fullname}
+					username={author.username}
```
Comment (spinningarrow) [inline src/components/KudosList.js]: Let's pass in the entire `author` object in the `author` prop - it seems more intuitive from an API point of view. Plus, we're using more than one of its properties so it makes sense.

**Comment by spinningarrow** [comment]
How about improving the look a little?
image
Maybe the author, kudos, and timestamp could be at the same level?

File: src/components/KudosNote.js (hunk 1 above)
Comment (xevenheaven) [inline src/components/KudosNote.js]: :joy:

File: src/components/KudosList.js (hunk 2 above)
Comment (xevenheaven) [inline src/components/KudosList.js]: Yes, that'll be good.

**Comment by xevenheaven** [comment]
Yup I'm planning to beef up the UI. Just first pass here! :yum:

**Comment by spinningarrow** [comment]
Okay, sounds good!

**Comment by xevenheaven** [comment]
How's the alignment now?

image

**Comment by spinningarrow** [comment]
Very nice!
You'll have to merge in manually though because it conflicts with the other PR I merged just now :P (#43)

**Pull request merged**